.luacheckrc

```diff
@@ -0,0 +1,242 @@
+codes = true
+std = "lua51"
```

Comment:
Let's use `std = "min"` as far as possible. There are a few files that need `std = "lua51"` because of setfenv, but most of our code should already be compatible with Lua 5.3 as soon as we throw out `module()`.

Reply:
`std = "min"` with additional globals `unpack`, `getfenv` and `setfenv`.

Reply:
Switched to `std = min` and add the globales `unpack`, `getfenv` and `setfenv`.